main/release/utils/github.py

```diff
@@ -71,8 +71,14 @@ def get_release_request(self) -> ReleaseRequest:
                 if re.compile("^([a-f0-9]{40})$").match(branch_name):
                     branch_name = DEFAULT_BRANCH
 
+            prefix = version_match.group("prefix")
+            if prefix:
+                # Required for sonar-scanner-azdo to support two artifacts in repox
+                # https://sonarsource.atlassian.net/browse/BUILD-5506
```

Comment:
There is no need to change this; only a remark: the ticket reference can be found in the commit, with a blame. Unless it's a `# TODO BUILD-5506 ...` to be later fixed, this is not useful as a comment. And it's also mentioned in the regex list.

The feature is generic, so the comment could be `# Allow to prefix the tag name` (and to suffix the project name?)

Reply:
This is a hacky way to support multiple artifacts, and is currently only used by one repo. If more repos need this, we need to come up with a better approach.